### ISSUE TITLE
Add support to Azure k8s cluster

### ISSUE THREAD

**Issue by katcipis** [NeowayLabs/klb#45]
There is a new cli made in python that can create k8s cluster integrated on Azure Container Services:

https://channel9.msdn.com/Shows/Azure-Friday/Kubernetes-Support-in-Azure-Container-Services

It seems really cool, although requires a new cli and it is a reasonably new feature. Perhaps it is a good idea to stay with the classic CoreOS (Linux Container ? :P) approach for now and use this later when we got things stable.

**Comment by vitorarins** [comment]
I don't see a problem in starting implementation of support for this service and I don't think it conflicts with giving the option of maintaining classic approaches. IMO `klb` would be much more complete when it supports all kinds of services from the main providers. So, as long as we keep things modularized, clean and organized, I say "the more the merrier" :)

**Comment by katcipis** [comment]
@vitorarins actually I was just making a point for not implementing this right away, it will stay on the backlog for now :-). The amount of time is finite ;-)

**Comment by vitorarins** [comment]
@katcipis I think accidentally we mixed two things here, which is the support for the new cli and the support for Azure Container Services. So I will open another issue for the new cli support.

**Comment by katcipis** [comment]
@vitorarins makes sense to have two issues, but they are related, this issue will depend on #85, and solving #85 will depend on building a strategy on how we will migrate/coexist both cli's (or not if we make an atomic upgrade).

**Comment by fabio-hara** [comment]
You can create another branch (using CLI 2.0) and start your tests...
...something like KLB 2 ?

**Comment by katcipis** [comment]
@fabio-hara definitely :+1:

**Comment by katcipis** [comment]
We now have support to using both azure cli 1.0 and 2.0
